mod/photos.php

```diff
@@ -1121,6 +1084,24 @@ function photos_content(App $a)
 			$pager->getItemsPerPage()
 		);
 
+		if ($cmd === 'drop') {
```

Comment:
Wouldn't be a `switch($cmd)` be better for this cases, would it?

Reply:
Maybe, but I'd rather not spend more time on this PR that I absolutely have too. It's already a pretty wide side track from #7068, the original PR.

Reply:
ok